database/tables/workspace_hosts.pg

```diff
@@ -1,8 +1,11 @@
 columns
+    became_unhealthy_at: timestamp with time zone
     hostname: text
     id: bigint not null default nextval('workspace_hosts_id_seq'::regclass)
     instance_id: text
+    launched_at: timestamp with time zone
     load_count: integer default 0
+    state: enum_workspace_host_state
```

Comment:
Maybe also columns for:
* `state_changed_at` -- last time we changed `state`
* `ready_at` -- first time we entered state "ready", so `ready_at - launched_at` tells us how long it took to launch
* `terminated_at`